src/test/java/core/basesyntax/dao/RegistrationServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationValidationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();

```

Comment:
Consider using `@BeforeEach` or `@BeforeAll` to initialize your `RegistrationServiceImpl` class instance. The `@BeforeEach` annotation is used when you want to initialize something before each test case, whereas `@BeforeAll` is used when you want to initialize something once before all tests run. Choose the one that is suitable for your test cases.

Reply:
fix comment